mlflow/ml-package-versions.yml

```
@@ -246,3 +246,22 @@ spacy:
     requirements: ["scikit-learn"]
     run: |
       pytest tests/spacy/test_spacy_model_export.py --large
+
+
+statsmodels:
+  package_info:
+    pip_release: "statsmodels"
```

```suggestion
    pip_release: "statsmodels"
    install_dev: |
      pip install git+https://github.com/statsmodels/statsmodels
```

Can we add `install_dev` to run the statsmodels tests using the dev version of statsmodels?